homework/basic_html_layout/feedback.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<style>
+    .parag1 {
+        padding: 0%;
+        margin-left: 40px;
+    }
+    .parag2 {
+        padding: 0%;
+        margin-left: 40px;
+    }
+
+    .wi {
+        padding-top: 40px;
+    }
+
+    button {
+        border-radius: 100px;
+    }
+</style>
+
+<body>
+    <h1>My feedback form</h1>
+    <form action="/action_page.php">
+        <ul>
+            <li>
+                <label for="li">Name</label>
+                <input type="text" name="li"><br>
+            </li>
+
+            <li>
+                <label for="li">Email:</label>
+                <input type="text" name="li"><br>
+            </li>
+
+            <li>
+                <label for="li">Password:</label>
+                <input type="text" name="li"><br>
+            </li>
+        </ul>
+
+        <ul>
+
+            <li>
+                Please check all the emotions that apply to you:
+            </li>
+            <div class="parag1">
+                <li><input type="checkbox" name="emotion1" value="angry">
```

Comment:
Use useful names here: `name="emotionAngry"`. Please change the label `for` accordingly.